models/comments.models.js

```diff
@@ -22,3 +34,11 @@ exports.insertNewComment = (articleId, newComment) => {
         return commentFromDB.rows[0];
     });
 };
+
+exports.removeComment = (commentId) => {
+    const queryString = `DELETE FROM comments WHERE comment_id=%L RETURNING *;`;
+
+    return db.query(format(queryString, commentId)).then(({ rows }) => {
+        return rows;
```

Comment:
We actually don't want to return rows here as we aren't sending a body back to the user

Reply:
I had to keep 'return rows' in the model, because otherwise my tests fail due to the if condition " if (deletedComment.length === 1)", which checks if there is a deleted comment and if there is, it sends the status 204 response. 

Please see the code below:

exports.deleteComment = (req, res, next) => {
    const { comment_id } = req.params;

    checkCommentIdExists(comment_id)
        .then(() => {
            return removeComment(comment_id);
        })
        .then((deletedComment) => {
            if (deletedComment.length === 1) {
                res.status(204).send();
            } else {
                return Promise.reject({ status: 404, msg: "Not found" });
            }
        })
        .catch((err) => {
            next(err);
        });
};